assets/app/scripts/controllers/services.js

```diff
@@ -24,7 +25,24 @@ angular.module('openshiftConsole')
       $scope.emptyMessage = "No services to show";
       updateFilterWarning();
       console.log("services (subscribe)", $scope.unfilteredServices);
-    }));    
+    }));
+
+    watches.push(DataService.watch("routes", $scope, function(routes){
+        $scope.routes = sortRoutes(routes._data);
```

Comment:
If sortRoutes is returning a map of routes grouped by service, then rename this to $scope.routesByService to be consistent with naming schemes in our other controllers